static/src/stylesheets/head.story-package.scss

```diff
@@ -0,0 +1,199 @@
+@import '_vars';
+@import '_mixins';
+
+@mixin kicker-override($tone, $colour) {
+    .tone-#{$tone}--item {
+        .fc-item__kicker,
+        .rich-link__kicker,
+        .fc-item__byline {
+            color: $colour;
+        }
+    }
+    
+    .tone-#{$tone}--sublink {
+        .fc-sublink__kicker {
+            color: $colour;
+        }
+    }
+}
+
+.fc-container--story-package {
+    background-color: colour(neutral-4);
+    margin-bottom: $gs-baseline;
+    border-top: 1px solid #ffffff;
+
+    // Container Chrome
+    .fc-container__header__title {
+        border-top-color: colour(story-package);
+        color: colour(news-support-2);
+
+        & a:hover {
+            color: colour(story-package);
+        }
+    }
+
+    .fc-container__header__description {
+        color: colour(story-package);
+    }
+
+    .treats__treat {
+        border-color: colour(news-support-2);
+        background-color: transparent;
+        color: colour(story-package);
+
+        &:hover {
+            background-color: inherit;
+        }
+    }
+
+    .button--primary {
+        background-color: colour(news-support-2);
+        border-color: colour(news-support-2);
+        color: #ffffff;
+
+        &:hover {
+            background-color: mix(colour(news-support-2), colour(neutral-1), 60%);
+            border-color: mix(colour(news-support-2), colour(neutral-1), 60%);
+        }
+
+        svg {
+            fill: #ffffff;
```

Comment:
If you remove the fill attribute within the SVG it will default to white in our new system. /cc @sndrs